src/types.ts

```diff
@@ -1,8 +1,13 @@
-const deprecateOkErr = () =>
+let deprecateOkErrShown = false;
+
+const deprecateOkErr = () => {
+  if (deprecateOkErrShown) return;
   console.warn(
     "The usage of the Ok/Err pattern is deprecated because of the danger of mismatched 'instanceof' types in node_modules. " +
       "Please use the regular try-catch pattern.",
   );
+  deprecateOkErrShown = true;
+};
```

Comment:
Should we eventually add a breaking change and remove this feature if it is deprecated? Should be a major version upgrade, but it is fine as we are the only ones using it, right?

Reply:
We need to remove all usages in all our projects first.  
Otherwise, we're going to lock ourselves from updates.